CMakeLists.txt

```diff
@@ -1,47 +1,66 @@
 cmake_minimum_required(VERSION 3.3)
-project(qmarkdowntextedit)
+project(qmarkdowntextedit LANGUAGES CXX)
 
 #set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
+#set(CMAKE_CXX_STANDARD 11) Use this instead
+#set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
 set(CMAKE_AUTOMOC ON)
 set(CMAKE_AUTOUIC ON)
 set(CMAKE_AUTORCC ON)
 set(CMAKE_INCLUDE_CURRENT_DIR ON)
 
-find_package( Qt5Core REQUIRED )
-find_package( Qt5Widgets REQUIRED )
-find_package( Qt5Gui REQUIRED )
+# add option to disable test executable
+option(QMARKDOWNTEXTEDIT_EXE "Build test executable" ON)
 
-qt5_wrap_ui(ui_qplaintexteditsearchwidget.h qplaintexteditsearchwidget.ui)
+# find qt
+find_package(QT NAMES Qt6 Qt5 REQUIRED COMPONENTS Widgets Svg)
+find_package(Qt${QT_VERSION_MAJOR} REQUIRED COMPONENTS Widgets Svg)
 
-set(RESOURCE_FILES
-        media.qrc
-        )
+# needed for windows
+if(WIN32)
+    set(INTL_LDFLAGS -lintl)
```

Comment:
Are you sure this is needed here?

Reply:
I don't have Windows and never tested it, but when you create a new CMake Project in Qt Creator, it's there. So no i'm not.